### DISCUSSION TITLE
Beatmap overlay missing time ago column in leaderboards

### DISCUSSION THREAD

**Discussion by Theighlin** [ppy/osu#16336] [Q&A] [answered]
game:
![image](https://user-images.githubusercontent.com/53081750/148259221-266949f7-4c71-499f-9fd6-35592c76e0a5.png)
web:
![image](https://user-images.githubusercontent.com/53081750/148259537-c41c0db8-400a-4c2f-82b7-3d5e05ed0387.png)

**Comment by peppy** [comment]
Why does it say "tempo"?

**Reply by smoogipoo** [reply]
Italian or some other localisation.

**Comment by bdach** [comment, marked as answer]
i've spent a bit of time looking at this today and as a result i got this:

![osu_2022-01-06_19-30-57](https://user-images.githubusercontent.com/20418176/148432901-1f50675a-3b35-4b75-afa4-6087d91fd09e.jpg)

now, it doesn't match web, but there are multiple disparate reasons for that:
* [web's setup](https://github.com/ppy/osu-web/blob/a8f5a68fb435cb19a4faa4c7c4bce08c4f096933/resources/assets/lib/scoreboard-time.tsx) is very coupled to the `momentjs` javascript library, to the point where the localisation analyser didn't pick up the [relevant localisations](https://github.com/ppy/osu-resources/blob/be13ff4ace496f7826a813a9743f88fdeb799dd9/osu.Game.Resources/Localisation/Web/CommonStrings.cs#L432-L490) as formattable strings, because they use js's `%d` placeholders rather than `:arg` laravel placeholders
* @peppy has expressed (in chat when i was streaming this) that web's formatting is bad/confusing (e.g. 3 months are shown as `3m`, which may well be interpreted as 3 minutes)
* my implementation additionally does not support localisations yet (would require https://github.com/ppy/osu-framework/pull/4918 at least)

do you think it's ok if i PR this? the code is available to preview over at https://github.com/ppy/osu/compare/master...bdach:scoreboard-time?expand=1.

**Reply by peppy** [reply]
I think this looks fine as a first implementation.
